workspace-types/src/lib.rs

```diff
@@ -28,3 +28,28 @@ impl BorshDeserialize for Raw {
         Ok(Self(res))
     }
 }
+
+#[derive(Debug, Clone, PartialEq, Eq, Serialize, Deserialize)]
+pub struct Raw20(pub [u8; 20]);
```

Comment:
Do we really need this type? 🤔

Reply:
No, this was removed before, please see above comment.

Reply:
You sure? 😄

Reply:
Same question.